source/_patterns/01-atoms/svgicon/demo/_svgicon-demo.scss

```diff
@@ -0,0 +1,15 @@
+// All component Sass needs non-printing base config
+@import '../../../00-protons/variables';
+@import '../scss/_icons-generated';
```

Comment:
Double-double verifying that this has **no** printing css in it.

Reply:
Protons is non-printing and icons-generated contains sass variables and methods to use the mixin.